lib/Comment/CommentResolveButton.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { bool, func } from 'prop-types';
+import Icon from '../Icon';
+import { MetaDot } from './CommentMeta';
+import Button from '../Button';
+
+export function CommentResolveButton({
```

Comment:
This is a new compound component, in the other PR it lived inside `Conversation.Header`, but now i've extracted it out so it's completely separate!